tool/update_bindings.dart

```diff
@@ -222,22 +222,19 @@ Future<void> _generateJsTypeSupertypes() async {
   }
 
   final jsTypeSupertypesScript = '''
-  // Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
-  // for details. All rights reserved. Use of this source code is governed by a
-  // BSD-style license that can be found in the LICENSE file.
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
 
-  // Updated by $_thisScript. Do not modify by hand.
+// Updated by $_thisScript. Do not modify by hand.
 
-  const Map<String, String?> jsTypeSupertypes = $jsTypeSupertypes;
-  ''';
+const Map<String, String?> jsTypeSupertypes = {
+${jsTypeSupertypes.entries.map((e) => "  ${e.key}: ${e.value},").join('\n')}
+};
+''';
   final jsTypeSupertypesPath =
       p.join(_bindingsGeneratorPath, 'js_type_supertypes.dart');
   await File(jsTypeSupertypesPath).writeAsString(jsTypeSupertypesScript);
-  await _runProc(
```

Comment:
Is the formatting done elsewhere?

Reply:
we're just emitting code now that's already formatted :)